app/scenes/Settings/Profile.js

```diff
@@ -44,8 +48,12 @@ class Profile extends React.Component<Props> {
     await this.props.auth.updateUser({
       name: this.name,
       avatarUrl: this.avatarUrl,
+      language: this.language,
     });
-    this.props.ui.showToast("Profile saved");
+    this.props.ui.showToast(t("Profile saved"));
+    setTimeout(function () {
+      window.location.reload();
+    }, 1000);
```

Comment:
If we use the hook from `react-i18next` then that should avoid the need to reload the entire app here, we can just call `changeLanguage` on save.

Reply:
done - changed to HOC *withTranslation*